devtools/debug-ferc1-etl.ipynb

```diff
@@ -156,8 +156,8 @@
     "\n",
     "def get_table_classes(module):\n",
     "    classes = [member[1] for member in inspect.getmembers(module, inspect.isclass)]\n",
-    "    table_classes = [x for x in classes if x.__name__.endswith(\"Ferc1TableTransformer\")]\n",
```

Comment:
@cmgosnell I ran into some FERC related errors I wasn't sure how to solve in the last two cells of this notebook.

Reply:
okay on the last cell i believe is a @katherinelamb / @zschira  question/ verification of my assumption: it looks like the new FERC plant classifier got pulled out of the transform step and thus this special exception of needing to pass in the fuel table into the steam tables' transform is no longer needed! so if that's true i think we can delete this cell and take out the `if table_name == "core_ferc1__yearly_steam_plants_sched402":` in the previous cell.

Reply:
and the second to last cell failed while validating the calculations in the table. the expected error rates were all set using full and fast etl settings and this notebook has a cell up top that set `years = [2020, 2021]` if you change it to `years = [2020, 2022]` and rerun the notebook this will work fine.

Reply:
@cmgosnell, yes that's true! `core_ferc1__yearly_steam_plants_sched402` is now handled like any other transform, and no longer has plant IDs

Reply:
Thanks y'all! Just made the changes.